third-party/realdds/include/realdds/topics/image-msg.h

```diff
@@ -31,6 +32,8 @@ namespace topics {
 
 class image_msg
 {
+    sensor_msgs::msg::Image _raw;
```

Comment:
This improves ROS compatibility? Were there image formats that were not supported before?

Reply:
This does not affect compatibility. Or maybe I misunderstand?